apps/aeutils/priv/rename_db

```diff
@@ -0,0 +1,17 @@
+-mode(compile).
```

Comment:
I've update these extensions scripts to work on Windows as well. Please ensure you  checked the updated scripts to ensure this new script works on Windows too. Also a respective `.cmd` file is needed in addition to the `.sh` file. And it must be added to `ci/appveyor/package.cfg`.

Reply:
I will try to prepare for Windows support today, so that you can review!

Reply:
@tolbrino https://github.com/aeternity/aeternity/pull/2196/commits/9282e86a94208edbb0247d96d35334298c34e385

Reply:
LGTM